docs/administration-guide/introduction.md

```diff
@@ -0,0 +1,34 @@
+# Administration Guide
+
+This guide is intended for more administrators of a Gaffer deployment and more
```

Comment:
```suggestion
This guide is aimed at administrators of a Gaffer deployment and more
```
Phrasing of this first sentence is a little weird and could do with a little rewrite :)